server/src/main/java/org/opensearch/action/search/UpdatePitContextResponse.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.action.search;
+
+import org.opensearch.common.io.stream.StreamInput;
+import org.opensearch.common.io.stream.StreamOutput;
+import org.opensearch.transport.TransportResponse;
+
+import java.io.IOException;
+
+/**
+ * Update PIT context response with creation time, keep alive etc.
+ */
+public class UpdatePitContextResponse extends TransportResponse {
+    private final String pitId;
+
+    private final long creationTime;
+
+    private final long keepAlive;
+
+    UpdatePitContextResponse(StreamInput in) throws IOException {
+        super(in);
+        pitId = in.readString();
+        creationTime = in.readLong();
+        keepAlive = in.readLong();
+    }
+
+    public UpdatePitContextResponse(String pitId, long creationTime, long keepAlive) {
+        this.pitId = pitId;
+        this.keepAlive = keepAlive;
+        this.creationTime = creationTime;
+    }
+
+    @Override
+    public void writeTo(StreamOutput out) throws IOException {
+        out.writeString(pitId);
```

Comment:
`super.writeTo(out)`?  (since we read from it here https://github.com/opensearch-project/OpenSearch/pull/2745/files#diff-73c304fb48b54c1fa28d5354de97e6d8aa338157deb9d307b991e299a8ccd5caR28)

Reply:
There is no super.writeto equivalent since there is no implementation in transport response.